x-pack/plugins/cases/public/components/markdown_editor/eui_form.tsx

```diff
@@ -60,6 +71,39 @@ export const MarkdownEditorForm = React.memo(
         [id, field.value, caseTitle, caseTags]
       );
 
+      useEffect(() => {
+        const storageDraftComment = draftCommentStorageKey && storage.get(draftCommentStorageKey);
+        if (storageDraftComment && storageDraftComment !== '') {
+          field.setValue(storageDraftComment);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+      }, []);
+
+      useEffect(() => {
+        if (initialValue && initialValue !== field.value) {
+          field.setErrors([{ message: i18n.COMMENT_VERSION_CONFLICT_ERROR }]);
```

Comment:
Thinking about this more, do we really need the user to refresh? We just want them to be aware that something changed under them. Maybe an alternative would be to some warning text that it change and saving will overwrite 🤷‍♂️

If we set the errors we still let the user save the change but nothing happens, I wonder if we should disable the save button if want to continue with this type of solution.

Reply:
I like the first approach, showing warning text instead of error as user is already in writing a comment and suddenly disabling the save button is a preventive action. So just a text warning suggesting that comment has changed makes sense.

Reply:
> So just a text warning suggesting that comment has changed, and letting user decide makes sense.

Sounds good to me!

Reply:
> > So just a text warning suggesting that comment has changed, and letting user decide makes sense.
> 
> Sounds good to me!

![image](https://user-images.githubusercontent.com/117571355/205352596-10202edd-6dfb-4b36-b294-e4130f3d4b25.png)